pkg/task/task.go

```diff
@@ -536,6 +551,22 @@ func (m *taskExecutor) putBatchV1Job() (err error) {
 	return
 }
 
+// putAppsV1STS will create a new StatefulSet
+// object in the cluster and store the response
+// in a json format
+func (m *taskExecutor) putAppsV1STS() (err error) {
+	j, err := m.asAppsV1STS()
```

Comment:
can we give a better name for this ?

Reply:
this is the name given across this file & package.
as per idiomatic maya, this should get into `pkg/kubernetes/statefulset/v1alpha1`